src/lib.rs

```diff
@@ -242,11 +243,11 @@ impl Lzxd {
     }
 
     /// Attempts to perform post-decompression E8 fixups on an output data buffer.
-    fn postprocess<'a>(
+    fn postprocess(
```

Comment:
I'm not sure why this changed. Was it an autofix from clippy perhaps?

Reply:
I got from idea: `Explicit lifetimes given in parameter types where they could be elided `  and it looks logical)